web_demo.py

```diff
@@ -196,19 +197,25 @@ def combine_history(prompt):
     total_prompt = total_prompt + cur_query_prompt.replace("{user}", prompt)
     return total_prompt
 
+def parse_args():
+    parser = argparse.ArgumentParser(description='Chat with the model, please add "--" between web_demo.py and Args in terminal')
+    parser.add_argument('--model_path', default='internlm/internlm-7b', help='path of the InternLm model')
+    parser.add_argument('--max_value', default=2048, help='the max length of the generated text')
+    return parser.parse_args()
 
-def main():
+def main(args):
     torch.cuda.empty_cache()
+    
     print("load model begin.")
-    model, tokenizer = load_model()
+    model, tokenizer = load_model(args.model_path)
     print("load model end.")
     
     user_avator = "doc/imgs/user.png"
     robot_avator = "doc/imgs/robot.png"
     
-    st.title("InternLM-Chat-7B")
+    st.title("InternLm-7b")
```

Comment:
```suggestion
    st.title("InternLM-Chat-7B")
```
This should be consistent with the model